src/main/java/mekanism/client/render/RenderTickHandler.java

```diff
@@ -304,6 +311,25 @@ public void onBlockHover(DrawSelectionEvent.HighlightBlock event) {
             MultiBufferSource renderer = event.getMultiBufferSource();
             Camera info = event.getCamera();
             PoseStack matrix = event.getPoseStack();
+
+            ItemStack stack = player.getMainHandItem();
+
+            // Draw outlines for area mining blocks
```

Comment:
I think it would be interesting/potentially useful to inside the if statement if we are rendering it add to vanilla's profiler (similar to what we do lower down) so that players can see using the debug pie chart if this ends up causing large percentage of the rendering time for specific setups.